src/msdo.ts

```diff
@@ -65,13 +65,38 @@ export class MicrosoftSecurityDevOps implements IMicrosoftSecurityDevOps {
         }
 
         let toolsString: string = core.getInput('tools');
+        let includedTools = [];
         if (!common.isNullOrWhiteSpace(toolsString)) {
             let tools = toolsString.split(',');
-            args.push('--tool');
             for (let i = 0; i < tools.length; i++) {
                 let tool = tools[i];
-                if (!common.isNullOrWhiteSpace(tool)) {
-                    args.push(tool.trim());
+                let toolTrimmed = tool.trim();
+                if (!common.isNullOrWhiteSpace(tool)
+                    && tool != "container-mapping"

```

Comment:
NIT: Use the enum value here 

```suggestion
                    && tool != Tools.ContianerMapping

```